Assignment2/bgDani-DanielBerrocal.ttl

```diff
@@ -1,10 +1,10 @@
 @base <http://www.oeg-upm.net/Organization/>
-@prefix pre: <http:/www.oeg-upm.net/Organization#/>
+@prefix pre: <http://www.oeg-upm.net/Organization#>
 
 :Class01            pre:includes:Sensor029;
                     pre:includes:Computer101.
 :Sensor029          pre:hasMeasurement:Measurement8401.
 :Computer101        pre:hasOwner:User10A.
-:Measurement8401    pre:hasTemperature:"29"^^xs:integer;
-                    pre:atTime:"2010-06-12T12:00:12"^^xs:date.
+:Measurement8401    pre:hasTemperature:"29"^^xsd:integer;
```

Comment:
Separate with a blank space the predicate and object. The same in the following line.